### PR TITLE
Implemented some @rmisio suggestions from #1419

### PR DESCRIPTION
**Pull request by Libter**
- https://github.com/OpenBazaar/OpenBazaar-Client/pull/1419#discussion_r60619540
- https://github.com/OpenBazaar/OpenBazaar-Client/pull/1419#discussion_r60621141

### CODE REVIEW

File: js/router.js [hunk 1]
```diff
@@ -48,9 +48,22 @@ module.exports = Backbone.Router.extend({
...
+        originalHistoryBack();
+    }
+
+    var originalHistoryForward = history.forward;
+    history.forward = function() {
+        historyAction = 'forward';
+        originalHistoryForward();
+    }
+    
+    window.historySize = -1;
```
Comment (jjeffryes) [inline js/router.js]: Why store these in the window? Can they be local values, like this.historySize?

File: js/router.js [hunk 2]
```diff
@@ -48,9 +48,22 @@ module.exports = Backbone.Router.extend({
         this.navigate(translatedRoute, { trigger: true });
       });
     });
-    history.size = -1;
-    history.position = -1;
-    history.action = 'default';
+    
+    var originalHistoryBack = history.back;
+    history.back = function() {
```
Comment (rmisio) [inline js/router.js:55]: It's slightly dangerous to overwrite such core functions. At the very least be sure to pass the arguments through and also maintain the return value. As of now, the core back and forward functions don't take any args, nor do they return anything, but we should make this as future-proof as possible:

``` javascript
var originalHistoryForward = history.forward;
history.forward = function() {
    historyAction = 'forward';
    return originalHistoryForward(arguments);
 }
```

**Comment by morebrownies** [comment]
If you guys could slip in a design change, I'd really appreciate it. Currently the entire button changes opacity to .3 when it's disabled. What I'd prefer to happen is only the text of the button changes opacity when it's disabled.

**Current:**
screenshot 2016-04-21 11 40 48

**What it should be:**
screenshot 2016-04-21 11 41 41

File: js/router.js (hunk 1 above)
Comment (Libter) [inline js/router.js]: Fixed in dc904b7c1cad9012b366012e28be9428294f6fb2.

File: js/router.js (hunk 2 above)
Comment (Libter) [inline js/router.js:55]: Fixed in c4bf3c0ecf20e1e79e3f28f2554bf77b456fe1a5.

**Comment by jjeffryes** [comment]
@morebrownies when this PR is ready, I'll take a look at the opacity. Currently the disabled class sets the opacity for the whole element, and sets pointer events to none. 

We could either set the disabled prop instead and a special opacity class, or create a text content only disabled class.

File: js/router.js [hunk 3]
```diff
@@ -108,28 +123,30 @@ module.exports = Backbone.Router.extend({
...
     else
         $('.js-navFwd').removeClass('disabled');
     
-    if (history.position == 1)
+    if (this.historyPosition == 1)
         $('.js-navBack').addClass('disabled');
     else
         $('.js-navBack').removeClass('disabled');
     
+    console.log('position: ' + this.historyPosition + '; size: ' + this.historySize);
```
Comment (jjeffryes) [inline js/router.js:148]: Don't forget to take this out.

**Pull request merged**